public/locales/de/translations.json

```diff
@@ -0,0 +1,49 @@
+	"roadmap.hard_fork.1": "Wie in der 'Soft Fork'-Phase wird weiterhin eine Ad-hoc-Managementstruktur zum Einsatz kommen, es wird jedoch mit der Einrichtung von 'Committees', 'Special Interest Groups' und 'Working Groups' begonnen. Demokratische Wahlen werden erst nach Abschluss dieser Phase durchgeführt um die nötigen Arbeiten zu erleichtern.",
+	"roadmap.hard_fork.2": "Zu diesem Zeipunkt werden wir ausreichend Mitwirkende haben, um NixPkgs (oder angemessene Teile davon) und Nix selbst sicher zu verwalten. Wir werden damit beginnen, Abweichungen vom 'Upstream Repository' zuzulassen und Änderungen an der Repository-Struktur zur Verbesserung der Entwicklung einzuführen.",
+	"roadmap.hard_fork.3": "Eine erste öffentliche Version des Paket-Repositorys wird erstellt und die Umbenennung `nix` auf `aux` gestartet, um Namesverwechselungen oder Fehler zu reduzieren.",
+	"roadmap.organization.title": "Strukturierung",
+	"roadmap.organization.1": "Aus dem Paket-Repository werden 'sets' extrahiert, und die Verwaltung an spezialisierte 'Sepcial Interest Groups' übergeben um langfristig eine einfachere Verwaltung zu ermöglichen. Beispielsweise werden `pythonPackages` in ein separates Repository verschoben, dessen Verwaltung in die Verantwortung der 'Python SIG' fällt.",
+	"roadmap.organization.2": "Die aktuelle 'flakes' Implementierung wird als v0 standardisiert. Die Funktion ist zwar nicht ideal, wird aber zu häufig genutzt, als dass sie geändert oder entfernt werden könnte, ohne das Ökosystem zu beinträchtigen. Stattdessen wird die v0-Implementierung aktiviert und zukünftige Arbeiten an Flakes werden von der  'Flakes SIG' übernommen.",
+	"roadmap.organization.3": "Die `aux` CLI wird verbessert, um eine ergonomischere Verwaltung von Paketen und Systemen zu ermöglichen. Zusätzliche Unterbefehle wie `aux system switch` und `aux system build` werden hinzugefügt, um das Onboarding und die laufende Wartung von Systemen zu erleichtern.",
+	"roadmap.infra.title": "Infrastruktur",
+	"roadmap.infra.1": "Da sich das Projekt nun erheblich vom Upstream-Projekt unterscheidet, müssen wir unsere eigenen 'Continuous Integration' und 'Binary Cache' Dienste bereitstellen. Bestehende Managementstrukturen werden genutzt, um die Einführung dieser Technologien zu verwalten.",
+	"roadmap.infra.2": "Die Dokumentation wird als Teil in den 'Continious Integration' Prozesses einbezogen und die 'Continous Deployment' wird so konfiguriert, dass sie auf dem neuesten Stand bleibt und es den Mitwirkenden erleichtert neue Lernmaterialien zu erstellen.",
+	"roadmap.infra.3": "Eine Suchinfrastruktur für Pakete und Optionen wird geschaffen.",
+	"roadmap.alignment.title": "Un­ab­hän­gig­keit",
+	"roadmap.alignment.1": "Wir haben es geschafft. Wir haben einen nachhaltigen und unabhängigen 'Fork' des Nix-Ökosystems geschaffen. Nachdem die ersten Arbeiten nun abgeschlossen sind, werden die erste demokratischen Wahlen abgehalten und gemeinsam werden wir neue Managementstrukturen aufbauen.",
```

Comment:
suggestion: Use 'Organisationsstrukturen' instead of 'Managementstrukturen', because under the 'Goals' headline we also chose 'Gesellschaftliche Organisation' as the translation of 'governance'.

Edit: Strg+F revealed this isn't the only place with 'Managementstrukturen'; amusingly I've only stumbled about it here 🤔

Reply:
Great suggestion! All changed.